cpp/unreal_projects/SpearSim/Config/DefaultInput.ini

```diff
@@ -77,7 +77,9 @@ bEnableFOVScaling=True
 bEnableLegacyInputScales=True
 bEnableMotionControls=True
 bFilterInputByPlatformUser=False
+bEnableInputDeviceSubsystem=True
```

Comment:
Did you add these explicitly or were they added automatically by Unreal?

Reply:
I did not add them, looks like Unreal adds them automatically when I setup input bindings in the editor.